vtkext/private/module/vtkF3DRenderer.cxx

```diff
@@ -1081,14 +1083,12 @@ void vtkF3DRenderer::ConfigureTextActors()
   {
     textColor[0] = textColor[1] = textColor[2] = 0.2;
   }
-  this->FilenameActor->GetTextProperty()->SetColor(textColor);
```

Comment:
no color control on the UI actor ?

Reply:
no need, it works well on all backgrounds since there is a dark local UI background too.